.github/workflows/node.js-ci.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ development ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: 14
+    - name: Install dependencies
+      run: npm ci
+    - name: Build Gatsby site
+      run: npm run build --if-present
+    - name: Deploy
+      uses: peaceiris/actions-gh-pages@v3
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        publish_dir: ./public
+        publish_branch: master  # default: gh-pages
```

Comment:
Since GITHUB_TOKEN is made available by default by a app installed on the repo when we enable Github Actions, maybe just doing `run: npm run deploy` should work as well.

Or if we explicitly need to pass in the GITHUB_TOKEN environment variable, we could do
```yml
run: npm run deploy
env:
  GITHUB_TOKEN : ${{ secrets.GITHUB_TOKEN }}
```

But, if it is already working this way, why to mingle with it, right 😁

Reply:
no the token does not work. We can remove the gh-pages  npm package since when running is not recognized as an action. If you try to git push with gh-pages using the action token, it does not work. git asks you to auth.